angrmanagement/ui/views/disassembly_view.py

```diff
@@ -153,6 +154,18 @@ def popup_xref_dialog(self, variable, asynch=True):
     # Public methods
     #
 
+    def subscribe_insn_select(self, callback):
+        """
+        Appends the provided function to the list of callbacks to be called when an instruction is selected in the
+        disassembly. The callback's parameters are:
+            'graph': the `QBaseGraph` object
+            'addr': integer address of the selected instruction
+            'block': the `QBlock` containing the instruction
+        :param callback: The callback function to call, which must accept **kwargs
+        """
+        self._linear_viewer.selected_insns.am_subscribe(callback)
+        self._flow_graph.selected_insns.am_subscribe(callback)
```

Comment:
Should these just be the same object? It would be really nice if the linear disassembly and the disassembly graph had the same selections when you toggle them

Reply:
The current implementation has `DisassemblyView` with two separate `QBaseGraph` objects, each with their own `selected_insns: set()`.  On select, `DisassemblyView` uses its `current_graph` property to figure out which graph is being used and only toggles that graph's selection.

`selected_insns` could be pulled up into `DisassemblyView` (or the `Workspace.instance`).